tests/backend/unit/services/messaging/test_messaging_service.py

```diff
@@ -7,12 +7,14 @@ class TestMessagingService(unittest.TestCase):
     def test_message_service_identifies_all_users(self):
         # Act
         usernames = MessageService._parse_message_for_username(
-            'Hello @[Iain Hunter] and "[LindaA1]'
+            'Hello @Iain Hunter and "@LindaA1 @@@Cersie @ @Peter_123'
         )
 
         # Assert
-        self.assertEqual(usernames[0], "Iain Hunter")
+        self.assertEqual(usernames[0], "Iain")
```

Comment:
Are we dropping support for usernames with a space in between? Like `Iain Hunter`?